jupyter-docker/scripts/run-jupyter.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Forces python 2
+JUPYTER_BASE="/usr/bin/python /usr/local/bin/jupyter-notebook"
+JUPYTER_CMD="$JUPYTER_BASE --NotebookApp.nbserver_extensions=\"{'jupyter_localize_extension':True}\" &> ${HOME}/jupyter.log"
```

Comment:
I had to add the `--NotebookApp.nbserver_extensions` to the jupyter command to make it load the `jupyter_localize_extension`. The same thing is also specified in `jupyter_notebook_config.py` but it wasn't getting loaded unless I explicitly added it here.